backend/src/main/java/com/ody/meeting/controller/MeetingController.java

```diff
@@ -68,4 +71,15 @@ public ResponseEntity<Void> validateInviteCode(
         return ResponseEntity.ok()
                 .build();
     }
+
+    @Override
+    @PatchMapping("/v1/meetings/{meetingId}/mate-statuses")
+    public ResponseEntity<MateStatusResponses> findAllMateStatuses(
+            @AuthMember Member member,
+            @PathVariable Long meetingId,
+            @RequestBody MateStatusRequest mateStatusRequest
+    ) {
+        MateStatusResponses mateStatuses = meetingService.findAllMateStatuses(meetingId, mateStatusRequest);
+        return ResponseEntity.ok(mateStatuses);
```

Comment:
[제안]

API 명세서 작성할 때 놓친 부분인데, `201`을 반환하는 건 어떤가요?
`PATCH`도 리소스 생성 메서드이니까요~!

```suggestion
        return ResponseEntity.status(HttpStatus.CREATED)
                .body(mateStatuses);
```

Reply:
`PUT` 이 자원이 저장하지 않을 경우 리소스를 생성하고
`PATCH`는 리소스의 일부만을 변경하기 때문에 201 응답보다는 
현재 응답 데이터도 있는 상황에선 200 응답이 적절하다고 생각합니다 🤔

Reply:
카키의 말처럼 PATCH는 기존 리소스를 수정하는 것이기 때문에 생성을 의미하는 201보다는 200이 적절할 것 같아요
꼼꼼하게 살펴보고 코드리뷰를 남겨주는 조조 좋아요 👍